src/transforms/eigenanalysis.jl

```diff
@@ -54,47 +66,46 @@ function apply(transform::EigenAnalysis, table)
 
   # original columns names
   cols = Tables.columns(table)
-  names = Tables.columnnames(cols)
+  onames = Tables.columnnames(cols)
 
   # table as matrix
   X = Tables.matrix(table)
 
-  # center the data
-  μ = mean(X, dims=1)
-  Y = X .- μ
```

Comment:
This operation is necessary inside EigenAnalysis. The other transforms PCA and SDS etc do additional steps, but this is irrelevant here.